cmake/CheckUncrustifyVersion.cmake

```diff
@@ -0,0 +1,13 @@
+if(UNCRUSTIFY_PRG)
```

Comment:
This script could possibly be generalized (I wonder if cmake has something like it already?)